res/database/Default/user/therob.xml

```diff
@@ -2412,18 +2725,22 @@
 			<title>
 				<de>%TITLE%</de>
 				<en>%TITLE%</en>
+				<pl>%TITLE%</pl>
 			</title>
 			<description>
 				<de>Familienkomödie über das Leben mit einem frechen Sohnemann. Das Drehbuch wird von dem Stardrehbuchschreiber Don Ron geschrieben.</de>
 				<en>Family comedy about life with a mischievous son. The script is written by star screenwriter Don Ron.</en>
+				<pl>Rodzinna komedia o życiu z psotnym synem. Scenariusz został napisany przez scenarzystę Dona Rona.</pl>
 			</description>
 			<variables>
 				<title>
 					<de>Engel oder Bengel|%NAME% dreht durch|%NAME% außer Rand und Band</de>
 					<en>Angel or Rascal|%NAME% goes berserk|%NAME% goes haywire</en>
+					<pl>Anioł lub łobuz|%NAME% - diabełek w owczej skórze|%NAME% rozrabia</pl>
 				</title>
 				<name>
 					<de>%PERSONGENERATOR_FIRSTNAME(de,1)%|%PERSONGENERATOR_FIRSTNAME(es,1)%|Tim</de>
+					<de>Pawełek|%PERSONGENERATOR_FIRSTNAME(es,1)%|Marcelek</de>
```

Comment:
wrong language tag - change to pl

Reply:
corrected.

Reply:
For now we do not have person generators (or "equivalents") for PL ..

these are "native" name generators
```
GetPersonGenerator().AddProvider( new TPersonGeneratorCountry_Austria )
GetPersonGenerator().AddProvider( new TPersonGeneratorCountry_Germany )
GetPersonGenerator().AddProvider( new TPersonGeneratorCountry_UK )
GetPersonGenerator().AddProvider( new TPersonGeneratorCountry_China	 )
GetPersonGenerator().AddProvider( new TPersonGeneratorCountry_Russia )
GetPersonGenerator().AddProvider( new TPersonGeneratorCountry_Turkey )
GetPersonGenerator().AddProvider( new TPersonGeneratorCountry_USA )
GetPersonGenerator().AddProvider( new TPersonGeneratorCountry_Denmark )
GetPersonGenerator().AddProvider( new TPersonGeneratorCountry_Greek )
GetPersonGenerator().AddProvider( new TPersonGeneratorCountry_Uganda )
GetPersonGenerator().AddProvider( new TPersonGeneratorCountry_Spain )
GetPersonGenerator().AddProvider( new TPersonGeneratorCountry_France )
```

For some countries we fall back to others (if not existing):
```
			case "SCO"        If HasProvider("UK") return "UK" else return ""
			case "E"          If HasProvider("UK") return "UK" else return ""
			case "IRL"        If HasProvider("UK") return "UK" else return ""
			case "NOR"        If HasProvider("DK") return "DK" else return ""
			case "SWE", "SE"  If HasProvider("DK") return "DK" else return ""
			case "SUI"        If HasProvider("AUT") return "AUT" else return ""
			case "BRA"        If HasProvider("ES") return "ES" else return ""
			case "POR"        If HasProvider("ES") return "ES" else return ""
			case "MEX"        If HasProvider("ES") return "ES" else return ""
			case "D"          If HasProvider("DE") return "DE" else return ""
```
If there is need we can always add some "Polish" ones. But then please also tell if we could use them for other countries too? so maybe Ukraine, Estonia, Latvia or whatever "might fit".
I understand that there are "country specific" names (especially name endings) but yeah, maybe we could come up with a nice list of "commonly used" ones which can be reused elsewhere.

here is a screenshot of the Russian name generator entries ... but I doubt we could reuse them here.
![image](https://github.com/TVTower/TVTower/assets/2625226/bd516ae2-c99c-4c1e-bdad-c5e576accf4d)


In my list of "German names" I eg. also have foreign names ("Abdulla") and more currently used versions formerly not "used so much" in Germany ("Zofia" instead of "Sof/phia")

Good sources are lists of "baby names" for your country ...and then taking 10-20 of each "popular letter" (you want more "Robert, Randy, Rico, Russel, Raymonds" than "Xavers and Xanthers" :D)

Reply:
Unfortunately, Polish names are different from Ukrainian, Estonian and Lithuanian names.
Ukrainian, Estonian match Russian more closely.

Reply:
As there is an upcoming PR for a Polish person generator, maybe you could make use of it already.
%PERSONGENERATOR_FIRSTNAME(pl,1)% for the first alternative.